scripts/rollup/results.json

```diff
@@ -1,17 +1,17 @@
 {
-  "branch": "master",
+  "branch": "warnForKeysInFragmentsRefactor",
```

Comment:
What's the policy for committing changes to `results.json`? It would be weird if `branch` was always just the name of the last branch merged into master.